src/packages/excalidraw/CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ Please add the latest change on the top under the correct section.
 
 ### Fixes
 
+- Changing z-index of elements (sorting them below/above other elements) now updates their `version` and `versionNonce` (only applies to the selected elements). This fix will allow you account for z-index changes if you're syncing just the elements that changed (and not the whole scene) [#3483](https://github.com/excalidraw/excalidraw/pull/3483).
```

Comment:
this should go under excalidraw library as it's related to the library and not API change?

Reply:
yes and no — this change is only useful to host maintainers so it'd not make much sense to add it under excalidraw library, no?

Reply:
hmm actually let it be, whatever affects host lets keep it under excal api only